sources/packages/web/src/components/institutions/request-a-change/InprogressOfferingChangeSummary.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <tab-container>
+    <body-header-container>
+      <template #header>
+        <body-header title="Applications" :recordsCount="applications?.count">
+          <template #subtitle>
+            Waiting for the student and StudentAid BC decision on the requested
+            change
+          </template>
+          <template #actions>
+            <v-text-field
+              density="compact"
+              label="Search name or application #"
+              variant="outlined"
+              v-model="searchCriteria"
+              @keyup.enter="searchApplicationOfferingChangeRecords"
+              prepend-inner-icon="mdi-magnify"
+              hide-details="auto"
+            >
+            </v-text-field>
+          </template>
+        </body-header>
+        <content-group>
+          <toggle-content :toggled="!applications?.count">
+            <v-data-table-server
+              :headers="InprogressOfferingChangeSummaryHeaders"
+              :items="applications?.results"
+              :items-length="applications?.count"
+              :loading="loading"
+              :items-per-page="DEFAULT_PAGE_LIMIT"
+              @update:options="paginationAndSortEvent"
+            >
+              <template #[`item.fullName`]="{ item }">
+                {{ item.columns.fullName }}
+              </template>
+              <template #[`item.studyStartPeriod`]="{ item }">
+                {{ dateOnlyLongString(item.columns.studyStartPeriod) }}
+                -
+                {{ dateOnlyLongString(item.value.studyEndPeriod) }}
+              </template>
+              <template #[`item.applicationNumber`]="{ item }">
```

Comment:
Not a part of the PR. Not sure why item is nor coming as typed. 
![image](https://github.com/bcgov/SIMS/assets/54600590/9547bc21-d4f3-4f9f-904c-3510c663ac29)

Reply:
are slots objects usually typed? I have never checked

Reply:
Me too. May be I haven't understood them very well.